tools/bazel.rc

```diff
@@ -38,3 +38,6 @@ build:clang-msan --copt -fsanitize=memory
 build:clang-msan --linkopt -fsanitize=memory
 build:clang-msan --define tcmalloc=disabled
 build:clang-msan --copt -fsanitize-memory-track-origins=2
+
+# Test options
+test --test_env=HEAPCHECK --test_env=PPROF_PATH
```

Comment:
Should this be `--test_env=HEAPCHECK=normal`? Is there a way to get the pprof location from the gperftools build in the local situation? Or if it's too complicated, what happens when you specify `HEAPCHECK=normal` but no `PPROF_PATH`?